beets/dbcore/query.py

```diff
@@ -447,6 +447,24 @@ def match(self, item):
         return any([q.match(item) for q in self.subqueries])
 
 
+class NotQuery(MutableCollectionQuery):
```

Comment:
Yeah, I can see why this was easy but might feel awkward. Probably the most straightforward approach is to not inherit from anything special (i.e., make Query itself the superclass). The `__init__` method would take a single subquery, and `clause` and `match` would work more or less as they do now.